### DISCUSSION TITLE
GitHub Issues

### DISCUSSION THREAD

**Discussion by Iarla-Monaghan** [community/community#114318] [New to GitHub]
To whom it may concern,
I have to use GitHub for a university assignment and have the following query: I created a quarto report in html format with various visualisations and when I rendered the document on posit cloud the visualisations appeared but when I uploaded the document to GitHub and published on the GitHub pages the visualisations did not appear. Is there any easy quick fixes because I am extremely busy at the moment and want this matter closed and I have limited experience with the GitHub platform.
Many thanks
From Iarla Monaghan

**Comment by devashish2024** [comment]
@Iarla-Monaghan Hey there, I'll be helping you.

![image](https://github.com/community/community/assets/83082760/7594a49e-68b0-4975-a294-182abe3448fe)
What is wrong in your README.md: you inputed the all html to input in index.html in your README.md.

> Solution: on your file README.md, click pencil, then rename the file from README.md to `index.html`
![image](https://github.com/community/community/assets/83082760/8b35b1db-964b-41e1-a515-c3e03be4f259)

![image](https://github.com/community/community/assets/83082760/39c747a8-89a8-4220-81a3-27b06d1db513)
This is good, no worries! 

> Your AC584-Quarto Assignment.html is visible at [here](https://iarla-monaghan.github.io/AC584-Quarto-Assignment/AC584-Quarto%20Assignment.html).

**Protip**: rename any of your HTML file to `index.html` and it will appear at https://iarla-monaghan.github.io/AC584-Quarto-Assignment/

**Comment by jfullstackdev** [comment]
good it's now live but you still have things to do like those files not being able to load, the path is the tricky one even if uploaded, so if you have further questions regarding it, you can still ask in here

![image](https://github.com/community/community/assets/47092464/55e58412-40e1-4ef4-935c-7504d5d355c8)
